pkg/cli/featureflag.go

```diff
@@ -0,0 +1,21 @@
+package cli
+
+import (
+	"github.com/spf13/pflag"
+	"github.com/spf13/viper"
+)
+
+const (
+	//RequiresAccessCode flag
+	FeatureFlagAccessCode string = "feature-flag-access-code"
+)
+
+// InitFeatureFlag initializes FeatureFlags command line flags
+func InitFeatureFlag(flag *pflag.FlagSet) {
```

Comment:
Usually this is pluralized:

```suggestion
func InitFeatureFlags(flag *pflag.FlagSet) {
```